### ISSUE TITLE
Dice values are so different.

### ISSUE THREAD

**Issue by WHloveHZN** [NicoloPecco/Swin-Functional-Fetal-Brain-Segmentation#1]
Hello, I would like to ask why there is such a big difference between validation and test Dice values. The validation set and test set are the same dataset.

**Comment by NicoloPecco** [comment]
Hi @WHLWJL, thanks for your comment! I'm not sure which specific values you're referring to. The validation set is used to fine-tune the model during training, while the **test set** is **completely unseen** by the model during training. Therefore Validation set performance might appear higher compared to the test set. Hope this clears up your doubt!

**Comment by WHloveHZN** [comment]
Thank you for your reply.
